addon/tailwind/components/bourbon-tooltip.css

```diff
@@ -23,18 +11,25 @@
   box-shadow: 1px 0 3px 2px var(--shade);
 }
 
-.theme-light .bourbon-tooltip__bg-default,
-.bourbon-tooltip__bg-default {
+.BourbonTooltip--light {
```

Comment:
I'm a little confused by how these classes interact: `.BourbonTooltip--light`, `.BourbonTooltip--dark`, `.theme-light`, `.theme-dark`.
Could you explain a bit?

Reply:
`.theme-light` and `.theme-dark` are coming from flabongo and how we used to figure out what color the tooltip should be before ie. a lighter one vs darker one.